packages/mui-material-nextjs/src/v13-pagesRouter/pagesRouterV13.tsx

```diff
@@ -113,6 +113,7 @@ export async function documentGetInitialProps(
   // You can consider sharing the same Emotion cache between all the SSR requests to speed up performance.
   // However, be aware that it can have global side effects.
   const cache = options?.emotionCache ?? createEmotionCache();
+  const { extractCriticalToChunks } = createEmotionServer(cache);
```

Comment:
If `createEmotionServer` is not called here there will be no `cache.compat = true` which ruins the server-side rendering.

Reply:
Let's add a comment before this line. It could be easy to break again in the future otherwise.

Reply:
```suggestion
  // The createEmotionServer has to be called directly after the cache creation due to the side effect of cache.compat = true,
  // otherwise the <style> tag will not come with the HTML string from the server.
  const { extractCriticalToChunks } = createEmotionServer(cache);
```

Reply:
In fact, it has to be
```
otherwise the <style> tag will come with the HTML string from the server.
```